pkg/logs/input/tailer/tailer.go

```diff
@@ -148,6 +152,7 @@ func (t *Tailer) forwardMessages() {
 		origin := message.NewOrigin(t.source)
 		origin.Identifier = identifier
 		origin.Offset = offset
+		origin.SetTags([]string{fmt.Sprintf("filename:%s", filepath.Base(t.path))})
```

Comment:
A tailer tails a single file, should we generate this tag at tailer startup and avoid doing it for each log line, or would it not matter?